### PR TITLE
Fix typos, formatting & minor wording issues

### PR DESCRIPTION
**Pull request by katrinleinweber**
Hi :-) While preparing [2019-06-26-blb-karlsruhe](https://github.com/vdb-org/2019-06-26-blb-karlsruhe) I noticed these minor issues.

Please feel free to leave review comments this week already. Some more commits may follow after the workshop / next weekend.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -4,24 +4,24 @@ title: Setup
...
 To participate in this Library Carpentry lesson, you will need a working UNIX-like shell environment.
-Specifically we will be using Bash ([Bourne Again Shell](https://en.wikipedia.org/wiki/Bash_(Unix_shell))) which is standard on Linux and Mac OS. 
-Even if you are a Windows user, learning Bash will open up a powerful set of tools on your personal machine, in addition to familiarizing you with the standard remote interface used on almost all servers and super computers. 
+Specifically we will be using Bash ([Bourne Again Shell](https://en.wikipedia.org/wiki/Bash_(Unix_shell))) which is standard on Linux and macOS.
+Even if you are a Windows user, learning Bash will open up a powerful set of tools on your personal machine, in addition to familiarizing you with the standard remote interface used on almost all servers and super computers.
 
 >## Terminal Setup
 >
->Bash is the default shell on most Linux distributions and Mac OS. 
+>Bash is the default shell on most Linux distributions and Mac OS.
```
Comment (libcce) [inline setup.md]: @katrinleinweber Still needs to be changed to macOS. I guess this means we will have to make this change throughout the lessons.

File: _episodes/05-counting-mining.md [hunk 2]
```diff
@@ -674,7 +674,7 @@ Pair up with your neighbor and work on these exercises:
 > > {: .bash}
 > >
 > > If you came up with something more advanced, perhaps including word boundaries,
-> > please share your result on the Etherpad and give yourself a pat on the shoulder.
+> > please share your result in the collaborative document and give yourself a pat on the shoulder.
```
Comment (libcce) [inline _episodes/05-counting-mining.md]: @katrinleinweber is collaborative document the preferred term? That makes sense and I wonder if we reference Etherpad in other places. Thanks for pointing this out. This is just a comment, I agree with the change.

**Comment by libcce** [comment]
@katrinleinweber only one change needs to be made, Mac OS to macOS. Otherwise, it is ready to merge 😄 Thanks!

File: _episodes/05-counting-mining.md (hunk 2 above)
Comment (katrinleinweber) [reply _episodes/05-counting-mining.md]: Not in the text anywhere, just in [`check_etherpad()`](https://github.com/LibraryCarpentry/lc-shell/blob/0e11b7e110b874264edf2d247646de2f8707c88d/bin/check-workshop#L237) which is [already updated in the `styles` repo](https://github.com/carpentries/styles/blob/6d015fe8f75ad7435a22aa8491dcfc66eed3ebb6/bin/workshop_check.py#L237), so I'm leaving that alone here.

**Comment by libcce** [comment]
Thanks @katrinleinweber 😄 I'll make that remaining change from Mac OS to macOS.

**Comment by katrinleinweber** [comment]
I've done that offline already and was waiting for the workshop to end. Please ask in the future, whether a draft PR is ready.

**Comment by libcce** [comment]
Most people have been experimenting with the feature, so sorry for not asking you. I'd sent a message earlier but you didn't respond. Was just trying to move this through quickly since the changes were small.

**Comment by katrinleinweber** [comment]
Which message was the earlier one? Your "Thanks" was in the exact same minute ;-) I can't reply that quickly while at a workshop.

"Small changes" which aren't labelled as a bug aren't so urgent IMHO, that the PR's maximum planned timeline of "next weekend" has to be overruled.

**Pull request merged**